### PR TITLE
feat: Add Helm chart and Kubernetes tutorial

### PR DESCRIPTION
**Pull request by rossf7**
Related to #29

This adds the Helm chart as well as a tutorial for installing it as well as Prometheus and Grafana. 

The dashboard file is pretty huge but here are the results from my test cluster.

scaphandre-kubernetes-dashboard

### CODE REVIEW

File: docs_src/tutorials/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+You can access the Prometheus web UI by creating a port forwarding connection.
+
+    kubectl port-forward deploy/prometheus-server 9090:9090
+
+## Install Grafana
+
+Create a configmap to store the Grafana dashboard.
+
+    kubectl create configmap scaphandre-dashboard \
+        --from-file=scaphandre-dashboard.json=docs_src/tutorials/grafana-kubernetes-dashboard.json
```
Comment (rossf7) [inline docs_src/tutorials/kubernetes.md:37]: This was the best way I could find to install the dashboard. Happy to move the files if they would be better somewhere else.

**Comment by bpetit** [comment]
Thanks a lot for the PR !
Don't worry about the failing test, my fix is waiting for another PR to be merged. I'll integrate those tests when I'll test the content of this one. This is great, thanks for your contribution :)

File: docs_src/tutorials/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+This tutorial uses [Helm](https://helm.sh/docs/intro/quickstart/) to install
+Scaphandre, Prometheus and Grafana.
+
+## Install Scaphandre
+
+First we install Scaphandre which runs as a daemon set which creates a pod on
+each node for collecting the metrics.
+
+    helm install scaphandre helm/scaphandre
```
Comment (bpetit) [inline docs_src/tutorials/kubernetes.md:11]: Does that suppose we rely on a local chart ? Maybe we should add the step to download the right chart on the right location first ?

File: docs_src/tutorials/kubernetes.md (hunk 1 above)
Comment (bpetit) [reply docs_src/tutorials/kubernetes.md:37]: Maybe we could find a better location later. I guess this is fine for now.

File: docs_src/tutorials/kubernetes.md (hunk 2 above)
Comment (rossf7) [reply docs_src/tutorials/kubernetes.md:11]: Yes its using the local chart but that's not ideal. 

The central Helm charts repo is now deprecated and not accepting new charts. https://github.com/helm/charts#helm-charts

The new pattern is to have a charts repo per org e.g. https://github.com/grafana/helm-charts and then link it to https://artifacthub.io/ for discoverability.

It should be possible to use something like GitHub Actions to push new versions of the chart to the repo. Happy to help with that if I can.

File: docs_src/tutorials/kubernetes.md (hunk 1 above)
Comment (rossf7) [reply docs_src/tutorials/kubernetes.md:37]: Yes I actually wanted to try loading the dashboard from Grafana Cloud here
https://grafana.com/grafana/dashboards/13845

But I ran into a bug with the Grafana chart.
https://github.com/grafana/helm-charts/issues/200

**Comment by bpetit** [comment]
I've tested it on a local cluster and the whole process works like a charm !

Thanks a lot. 😃 

I'll integrate the recent fix I made for the CI job in a local branch and merge from that.

File: docs_src/tutorials/kubernetes.md (hunk 2 above)
Comment (bpetit) [reply docs_src/tutorials/kubernetes.md:11]: This is fine as a first version.

We may create an FR after that to setup a helm chart repository for scaphandre.

**Pull request merged**